extension-style/src/main/java/com/mapbox/maps/extension/style/utils/TypeUtils.kt

```diff
@@ -185,6 +185,29 @@ fun Value.unwrapToStyleTransition(): StyleTransition {
   throw UnsupportedOperationException("unable to unwrap to StyleTransition: $this")
 }
 
+/**
+ * Unwraps an literal array to a array.
+ *
+ * e.g. translate ["literal", [0.0, 0.0, 0.0]] to [0.0, 0.0, 0.0].
+ */
+internal fun Expression.unwrapFromLiteralArray(): Expression {
+  if (this.contents is List<*>) {
+    @Suppress("UNCHECKED_CAST")
+    val listValue = this.contents as List<Value>
+    val operator = listValue.first().contents as? String
+    if ("literal" == operator) {
```

Comment:
just my small imo: `ourVariable == "literal"` looks a bit more familiar 😄

Reply:
I think it's a habbit from Java world, where avoid mistype "==" with "=", but in this case, I guess it doesn't really matter.